test/stripe/charge_test.rb

```diff
@@ -40,6 +40,16 @@ class ChargeTest < Test::Unit::TestCase
       assert charge.is_a?(Stripe::Charge)
     end
 
+    context ".capture" do
+      should "capture the charge" do
+        charge = Stripe::Charge.capture("ch_123", amount: 100)
+        assert_requested :post,
+                         "#{Stripe.api_base}/v1/charges/ch_123/capture",
+                         body: { amount: 100 }
+        assert charge.is_a?(Stripe::Charge)
+      end
+    end
+
```

Comment:
Apparently we never tested `capture` before? Added a test.

Reply:
Yeah, I noticed that we had a few missing tests too.